app/src/main/java/ch/epfl/sdp/appart/database/FirestoreDatabaseService.java

```diff
@@ -61,6 +62,17 @@
     @Inject
     public FirestoreDatabaseService() {
         db = FirebaseFirestore.getInstance();
+        /*
+            Do we use this or not ?
+            If we do, this "overrides" or local DB system. However, we know that the data will
+            have the same version that our local db. So maybe, we can think of the local db
+            as a backup for data that can't be cached by firestore.
+            
+            db.clearPersistence();
+            FirebaseFirestoreSettings settings = new FirebaseFirestoreSettings.Builder()
+                .setPersistenceEnabled(false).build();
+            db.setFirestoreSettings(settings);
+        */
```

Comment:
I think it's a good choice to let Firebase cache, but it will definitely be useful when testing the localdb so i'd say leave it there commented out